tests/test_iterables.py

```diff
@@ -0,0 +1,116 @@
+from algorithms.iterables import (
+    convolved
+)
+
+import unittest
+
+
+class TestConvolved1D(unittest.TestCase):
```

Comment:
the unit tests for `conv` should have been covered within the `conv` library. so this is invalid

Reply:
Sure, those tests are the same as in the library, and copied. So we should just remove them from here?